extensions/azurePublish/src/components/ChooseProvisionAction.tsx

```diff
@@ -0,0 +1,264 @@
+  padding: 20px;
+`;
+
+const Title = styled(Text)`
+  font-size: ${FluentTheme.fonts.xLarge.fontSize};
+  margin: 8px 0;
+`;
+
+const Summary = styled(Stack)`
+  margin: 8px 0;
+`;
+
+const Details = styled(Stack)`
+  margin: 10px 0;
+`;
+
+const Instruction = styled(Stack)`
+  margin: 10px 0;
+`;
+
+const InstructionTitle = styled(Text)`
+  font-size: ${FluentTheme.fonts.smallPlus.fontSize};
+  text-transform: uppercase;
+  margin: 8px 0;
+`;
+
+const InstructionDetails = styled.div`
+  margin: 10px 0;
+`;
+
+const ResourceTitle = styled(Stack)`
+  margin: 10px 0;
+`;
+
+const LearnMoreLink = styled(Link)`
+  user-select: none;
+`;
+
+// ---------- CreateActionContent ---------- //
+
+const CreateActionContent = () => {
+  return (
+    <Content>
+      <Title>{formatMessage('Create New Resources')}</Title>
+      <Summary>
+        <Text>
+          {formatMessage(
+            'Select this option when you want to provision new Azure resources and publish a bot. A subscription to Microsoft Azure is required.'
+          )}
+        </Text>
+      </Summary>
+      <Details>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 1')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>{formatMessage('Sign in to Azure.')}</Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 2')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>
+              {formatMessage(
+                'Select tenant & subscription, enter resource group name and resource name, and select region.'
+              )}
+            </Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 3')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>
+              {formatMessage(
+                'Review & create new resources. Once provisioned these resources will be available in your Azure portal.'
+              )}
+            </Text>
+          </InstructionDetails>
+        </Instruction>
+      </Details>
+      <LearnMoreLink>{formatMessage('Learn More')}</LearnMoreLink>
+    </Content>
+  );
+};
+
+const ImportActionContent = () => {
+  return (
+    <Content>
+      <Title>{formatMessage('Import existing resources')}</Title>
+      <Summary>
+        <p>
+          <Text>{formatMessage('Select this option to import existing Azure resources and publish a bot.')}</Text>
+        </p>
+        <p>
+          <Text>
+            {formatMessage(
+              'Edit the JSON file in the Publish Configuration field. You will need to find the values of associated resources in your Azure portal. A list of required and optional resources may include:'
+            )}
+          </Text>
+        </p>
+      </Summary>
+      <Details>
+        <ResourceTitle>
+          <Text>Microsoft Application Registration</Text>
```

Comment:
formatMessage on these?

Reply:
They are product names and the machine translation will change Azure to blue.